addons/danger/stringtable.xml

```diff
@@ -3,84 +3,111 @@
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_DisplayName">
             <English>Configure Group AI</English>
+            <German>Gruppen-KI einstellen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_SettingName">
             <English>Disable LAMBS group AI</English>
+            <German>LAMBS KI für Gruppe ausschalten</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Module_DisableGroupAI_SettingToolTip">
             <English>Disables LAMBS group AI\nDisabling this feature prevents autonomous building assaults and clearing, as well as hiding from aircraft and tanks</English>
+            <German>Das Ausschalten verhindert das autonome Stürmen und Säubern von Gebäuden sowie das verstecken vor Panzern und Flugobjekten.</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_MainCat">
             <English>Settings</English>
+            <German>Einstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_GeneralCat">
             <English>General</English>
+            <German>Allgemein</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_ShareInformationCat">
             <English>Settings Share information</English>
+            <German>Informationen-Teilen Einstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_CQBFormationsCat">
             <English>Settings CQB Formations</English>
+            <German>CQB-Formationseinstellungen</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_Debug">
             <English>Debug</English>
+            <German>Debug</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableDangerFSM">
             <English>Disable danger.fsm for player group</English>
+            <German>Danger.fsm für Spielergruppe deaktivieren</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableDangerFSM_ToolTip">
             <English>Toggle advanced danger.fsm features on player group</English>
+            <German>Erweiterte danger.fsm Funktionen für Spielergruppe umschalten</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableSuppressionPlayerGroup">
             <English>Disable suppression from player group</English>
+            <German>Niederhaltung für Spielergruppe deaktivieren</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableSuppressionPlayerGroup_ToolTip">
             <English>Toggle autonomous suppression for units in player group</English>
+            <German>Autonome Niederhaltung für Einheiten der Spielergruppe umschalten</German>
         </Key>
         <Key ID="STR_Lambs_Danger_Settings_DisableReactPlayerGroup">
             <English>Disable reaction state on player group</English>
+            <German>'Reaction State' für Spielergruppe ausschalten</German>
```

Comment:
`Reaction State` --> `Reaktionszustand`
Hintergrund ist ein "zustand" in der FSM Statemachine die "Reaction" heißt.

Reply:
Fixed